### ISSUE TITLE
CreateProcess() returned 5

### ISSUE THREAD

**Issue by TemporalAgent7** [alinmear/docker-conanexiles#58]
Hi,

I'm trying to set this up on my Docker host (a Windows box running Docker Desktop under WSL2). If I open port 5900 and use a VNC client to connect, I'm seeing this error message:

![image](https://user-images.githubusercontent.com/48541170/154199641-c494d5db-2381-4b96-880b-bc9d27eb4492.png)

Pressing OK just terminates the process I presume, and in a few seconds I see the same error message being displayed.

There's not much in the logs, they end like this:

```
max open descriptors set to 2048

2022-02-16 05:58:16,484 INFO spawned: 'conanexilesServer' with pid 1502

2022-02-16 05:58:17,486 INFO success: conanexilesServer entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)

conanexilesServer: started

max open descriptors set to 2048

max open descriptors set to 2048

max open descriptors set to 2048

max open descriptors set to 2048

max open descriptors set to 2048

2022-02-16 06:04:24,974 INFO exited: conanexilesServer (exit status 45; not expected)

max open descriptors set to 2048

2022-02-16 06:04:49,122 INFO spawned: 'conanexilesServer' with pid 1837

2022-02-16 06:04:50,124 INFO success: conanexilesServer entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)

conanexilesServer: started
```

I basically set this up exactly as in the checked in docker-compose.yml (same environment variables, same open ports, etc.). Please let me know if there's any additional information I can collect or any steps I can take to troubleshoot.

Thank you!

**Comment by flogie** [comment]
Running into the same Problem. Have you (or anybody) been able to find a solution to this problem?

**Comment by TemporalAgent7** [comment]
No. I ended up running it on bare metal (not Docker) for a while, but since moved on to other games.

**Comment by flogie** [comment]
Seems to be caused by running the container with a bindmount to a Windows Filesystem on Docker Desktop. 

When using a Docker Volume the container starts normal.

**Comment by xxxsen** [comment]
similar issue, but i got exit status 5 in terminal, not 45
